sceptre/cli/helpers.py

```diff
@@ -89,9 +89,20 @@ def write(var, output_format="str", no_colour=True):
     click.echo(stream)
 
 
+def _get_config_reader(ctx):
```

Comment:
So about to put a PR in that simplifies that CLI commands to just use the `get_stack_or_env` function. So what I suggest is you can probably remove this function and the tests and then rebase of the new master.

Reply:
#294